### ISSUE TITLE
handle URLs without http prefix

### ISSUE THREAD

**Issue by arlin** [phylotastic/phylotastic-portal#237]
Users might enter a URL without the http:// prefix, e.g., www.eol.org.  The portal does not recognize this and returns an error, with no explanation other than "Cannot process this link". 

Change the message to "Invalid URL.  Did you forget the http prefix?  To be sure, go to the page you want, then copy the text from the browser's URL bar."

**Comment by epontell** [comment]
should be easy to parse the url to see if it starts with http: or https: (if not, add it).

**Comment by arlin** [comment]
Yes, I agree.  If URL does not start with "http", prepend "http://".   Change the error message to "Invalid URL.  To ensure success, go to the page you want, then copy the text from the browser's URL bar."

**Comment by arlin** [comment]
@ducvan0212 , are you attempting to detect and fix a missing "http://"?  Or is it just the message.  I will close this if you are only doing the message.  However, I tried a URL missing the prefix and it doesn't work.

**Comment by ducvan0212** [comment]
<img width="668" alt="screen shot 2016-11-03 at 9 24 03 am" src="https://cloud.githubusercontent.com/assets/1614884/19972338/58491db0-a1a7-11e6-976a-01258dc90965.png">

Because the prefix maybe https or http so we dont know which one will work for url provided. I think it is better to let user copy and then paste the link to portal.

**Comment by arlin** [comment]
Can we just make the message a bit more informative for the user?  I suggest this: 

"Invalid URL. To ensure success, go to the page you want, then select-all and copy from the browser's URL bar into the URL box below. "
